client/src/main/scala/drt/client/components/MultiDayExportComponent.scala

```diff
@@ -76,18 +76,24 @@ object MultiDayExportComponent {
                 }),
                 <.div(
                   <.div(^.className := "multi-day-export-links",
-                    <.a("Export Arrivals",
-                      ^.className := "btn btn-default",
-                      ^.href := SPAMain.absoluteUrl(s"export/arrivals/${state.startMillis}/${state.endMillis}/${props.terminal}"),
-                      ^.target := "_blank",
-                      ^.onClick -->{Callback(GoogleEventTracker.sendEvent(props.terminal, "click", "Export Arrivals", f"${state.startYear}-${state.startMonth}%02d-${state.startDay}%02d - ${state.endYear}-${state.endMonth}%02d-${state.endDay}%02d"))}
-                    ),
-                    <.a("Export Desks",
-                      ^.className := "btn btn-default",
-                      ^.href := SPAMain.absoluteUrl(s"export/desks/${state.startMillis}/${state.endMillis}/${props.terminal}"),
-                      ^.target := "_blank",
-                      ^.onClick -->{Callback(GoogleEventTracker.sendEvent(props.terminal, "click", "Export Desks", f"${state.startYear}-${state.startMonth}%02d-${state.startDay}%02d - ${state.endYear}-${state.endMonth}%02d-${state.endDay}%02d"))}
-                    )
+                    if (props.loggedInUser.hasRole(ArrivalsAndSplitsView))
+                      <.a("Export Arrivals",
+                        ^.className := "btn btn-default",
+                        ^.href := SPAMain.absoluteUrl(s"export/arrivals/${state.startMillis}/${state.endMillis}/${props.terminal}"),
+                        ^.target := "_blank",
+                        ^.onClick --> {
+                          Callback(GoogleEventTracker.sendEvent(props.terminal, "click", "Export Arrivals", f"${state.startYear}-${state.startMonth}%02d-${state.startDay}%02d - ${state.endYear}-${state.endMonth}%02d-${state.endDay}%02d"))
+                        }
+                      ) else EmptyVdom,
+                    if (props.loggedInUser.hasRole(ArrivalsAndSplitsView))
```

Comment:
should the desks export have a more semantic role?

Reply:
good point, this is actually part of the next ticket - updating the UI, but I started working on it here before we'd made this ticket, I'll fix this part for now!